src/htmlminifier.js

```diff
@@ -662,6 +662,33 @@ function uniqueId(value) {
   return id;
 }
 
+function weightAttributes(value) {
```

Comment:
I think the name is a bit confusing for a function that creates another function. Maybe something like "makeAttrSorter"? Although that sounds awful too...